node/reqres.js

```diff
@@ -75,9 +86,18 @@ function TChannelOutgoingRequest(id, options, sendFrame) {
     self.sendFrame = sendFrame;
 }
 
-TChannelOutgoingRequest.prototype.send = function send(arg1, arg2, arg3) {
+inherits(TChannelOutgoingRequest, EventEmitter);
+
+TChannelOutgoingRequest.prototype.send = function send(arg1, arg2, arg3, callback) {
     var self = this;
     self.sendFrame(arg1, arg2, arg3);
+    if (callback) {
+        callback = once(callback);
```

Comment:
suggestion: use a technique like ( https://github.com/tommymessbauer/hammock/blob/master/lib/response.js#L32-L39 )

This means you will remove listeners and not leave dangling error or response listeners on the `TChannelOutgoingRequest` instance.